jdaviz/configs/specviz/plugins/viewers.py

```diff
@@ -65,6 +65,11 @@ def on_mouse_or_key_event(self, data):
             else:  # pragma: no cover
                 return
 
+        # only handle mouseover markers if the setting is selected:
+        plot_options_plugin = self.jdaviz_app.get_tray_item_from_name('g-plot-options')
+        if not plot_options_plugin.show_mouseover_marker:
+            return
```

Comment:
I think we want only the marker to be toggled, but still show the information in the panel.  So `self.label_mouseover.marks[self._reference_id].visible = plot_options_plugin.show_mouseover_marker` somewhere.

Reply:
right.  Alternatively, the switch could act as the current `locking_active` (but probably renamed), and the text in the plugin changed to reflect that.  That actually would probably be my vote (the switch controls both the marker AND exposing the coordinates of the nearest spectrum, but the mouse position is always exposed).